pkg/api/router.go

```diff
@@ -85,6 +85,17 @@ func (s *server) setupRouting() {
 		),
 	})
 
+	handle(router, "/pss/send/{topic}/{targets}", jsonhttp.MethodHandler{
+		"POST": web.ChainHandlers(
+			jsonhttp.NewMaxBodyBytesHandler(swarm.ChunkSize),
+			web.FinalHandlerFunc(s.pssPostHandler),
+		),
+	})
+
+	handle(router, "/pss/subscribe/{topic}", jsonhttp.MethodHandler{
+		"GET": http.HandlerFunc(s.pssWsHandler),
```

Comment:
this should also be `POST` no? 
you  convinced me that there  is no  need for `DELETE` to cancel a subscription

Reply:
> this should also be POST no?

Tried it. Didn't work. Also the websocket library does not offer any notion of HTTP verbs. Since WS is an actual protocol, I would not assume that it behaves as HTTP does.

Reply:
?? I mean creating   thewwebsocket should be 

     POST /pss/subscribe/{topic}

Reply:
because when you create the websocket you don't `POST`. As a client you don't even use any HTTP notions at all, you dial to `ws://...`, you don't `HTTP POST /path/to/endpoint` and magically get a websocket back

Reply:
as of such, you dont even need to mention the HTTP verb in the book spec. You just specify `ws://localhost:8080/pss/subscribe/<topic>` and that's that

Reply:
`jsonhttp.MethodHandler` can be even removed here.

Reply:
i understood that this is an http API endpoint. otherwise what are the http-s all  around it?  i  must be missing  something, but  i should  do my homework first sorry guys for interfering